### PR TITLE
Add absorbed model serialisation

### PR DESCRIPTION
**Pull request by QRemy**
add specific from_dict() and to_dict() for Absorption and AbsorbedSpectralModel

### CODE REVIEW

File: gammapy/modeling/models/__init__.py [hunk 1]
```diff
@@ -38,5 +37,6 @@
     "TableModel": TableModel,
     "SpectralGaussian": SpectralGaussian,
     "SpectralLogGaussian": SpectralLogGaussian,
-    "ScaleModel": ScaleModel,
+    "AbsorbedSpectralModel":AbsorbedSpectralModel,
```
Comment (cdeil) [inline gammapy/modeling/models/__init__.py]: Please run `make polish` to auto-format this code properly (missing space).

**Comment by QRemy** [comment]
It seems that bins edges are given only for energy in the file, as I didn't know if there is a reason or convention to explain that I just adapted the read method for now.

**Comment by registerrier** [comment]
> @QRemy - Thanks!
> 
> Looks good to me.
> 
> I presume changing the FITS format to just have node positions instead of the annoying and useless edge positions for energy and param isn't something you want to do here?

the format is the one used by the XSpec table models. It might be wise to keep it for FITS import.

**Comment by cdeil** [comment]
Discussed with @registerrier and @adonath just now - we're not sure it's a good idea to have a second YAML serialisation in addition to the existing FITS serialisation, especially given that it's slightly inconsistent with one using bin edges and the other bin centers.

But since we're not sure what we want here, and this is working code and nice functionality, I'll merge this in now. Possibly something we re-dicuss in the future ...

**Pull request merged**